src/EnergyPlus/DualDuct.cc

```diff
@@ -268,125 +245,111 @@ namespace DualDuct {
+                                                         DataLoopNode::NodeFluidType::Air,
+                                                         DataLoopNode::ConnectionType::Outlet,
+                                                         NodeInputManager::CompFluidStream::Primary,
+                                                         DataLoopNode::ObjectIsNotParent,
+                                                         cAlphaFields(3));
+                thisDD.HotAirInletNodeNum = GetOnlySingleNode(state,
+                                                              AlphArray(4),
+                                                              ErrorsFound,
+                                                              DataLoopNode::ConnectionObjectType::AirTerminalDualDuctConstantVolume,
+                                                              AlphArray(1),
+                                                              DataLoopNode::NodeFluidType::Air,
+                                                              DataLoopNode::ConnectionType::Inlet,
+                                                              NodeInputManager::CompFluidStream::Primary,
+                                                              DataLoopNode::ObjectIsNotParent,
+                                                              cAlphaFields(4));
+                thisDD.ColdAirInletNodeNum = GetOnlySingleNode(state,
+                                                               AlphArray(5),
+                                                               ErrorsFound,
+                                                               DataLoopNode::ConnectionObjectType::AirTerminalDualDuctConstantVolume,
+                                                               AlphArray(1),
+                                                               DataLoopNode::NodeFluidType::Air,
+                                                               DataLoopNode::ConnectionType::Inlet,
+                                                               NodeInputManager::CompFluidStream::Primary,
+                                                               DataLoopNode::ObjectIsNotParent,
+                                                               cAlphaFields(5));
+
+                thisDD.MaxAirVolFlowRate = NumArray(1);
+                thisDD.ZoneMinAirFracDes = 0.0;
 
                 // Register component set data - one for heat and one for cool
-                TestCompSet(
-                    state, CurrentModuleObject + ":HEAT", state.dataDualDuct->dd_airterminal(DDNum).Name, AlphArray(4), AlphArray(3), "Air Nodes");
-                TestCompSet(
-                    state, CurrentModuleObject + ":COOL", state.dataDualDuct->dd_airterminal(DDNum).Name, AlphArray(5), AlphArray(3), "Air Nodes");
+                BranchNodeConnections::TestCompSet(state, CurrentModuleObject + ":HEAT", thisDD.Name, AlphArray(4), AlphArray(3), "Air Nodes");
```

Comment:
I think in this specific case, i.e, string + (const)string, `format` offers no advantage.